backend/locations/v1/buildings/sqlmigration.go

```diff
@@ -0,0 +1,123 @@
+package buildings_v1
+
+// import MSSQL and PSQL drivers and libraries that allow us to connect
+import (
+	"database/sql"
+	"log"
+
+	_ "github.com/denisenkom/go-mssqldb"
+	_ "github.com/lib/pq"
+)
+
+func migrate() {
```

Comment:
Lets add a todo here.

```suggestion
//TODO(@ronydahdal) finish :)
func migrate() {
```